middleware/CompressionHandler.md

```diff
@@ -1,47 +1,64 @@
 # Compression Handler
 
-A middleware component that requests, detects and decompresses response bodies.
+A middleware component that compresses request bodies and falls back to an uncompressed body if the API doesn't support request body compression.
 
 ## Requirements
 
-- Add `Accept-encoding: gzip` to request headers.
-- Decompress response bodies that have the header `Content-Encoding: gzip`
+- Add `Content-encoding: gzip` to request headers for requests with a body.
```

Comment:
Does AGS only support gzip?

Reply:
from the information they shared with me by email, yes, that's the only compression format they will support for now. Brotli would have been nice.